base/applications/network/net/main.c

```diff
@@ -45,6 +46,24 @@ COMMAND cmds[] =
 
 HMODULE hModuleNetMsg = NULL;
 
+VOID PrintTranslatedStatus(
+    NET_API_STATUS status)
+{
+    switch (status)
+    {
+        case NERR_Success:
+            ConResPuts(StdOut, IDS_SUCCESS);
+            break;
+        case ERROR_ALIAS_EXISTS:
+           ConResPuts(StdOut, IDS_USEREXISTS);
+           break;
+        case NERR_UserNotFound:
+            ConResPuts(StdOut, IDS_NOTFOUND);
+            break;
+        default:
+            PrintMessageString(status);
```

Comment:
ok so just to be sure, does it correctly behave as expected?

Reply:
@HBelusca Can we can use `PrintMessageString` insted of my `PrintTranslatedStatus` function?
I tested this and found we don't have Russian translation for `PrintMessageString`.

Reply:
@nagayev I just looked into it, so `PrintMessageString` function calls `FormatMessage`, which uses `netmsg.dll` as the resource library for localized messages.

So that means the way you do translation here is not correct, you should modify this file instead: https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/mc/netmsgmsg.mc;hb=b1fca7ea557f079c3c674f7d6eb5217b1c559d08